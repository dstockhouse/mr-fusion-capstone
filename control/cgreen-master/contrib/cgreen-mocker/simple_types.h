int return_int(void);
char return_char(char c);
char* return_pointer_to_char(char *string);
