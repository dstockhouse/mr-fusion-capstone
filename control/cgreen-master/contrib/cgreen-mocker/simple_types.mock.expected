/* -*- c -*-*/
#include "simple_types.h"
#include <cgreen/mocks.h>

int return_int(void) { 
  return (int) mock();
}

char return_char(char c) { 
  return (char) mock(c);
}

char *return_pointer_to_char(char *string) { 
  return (char *) mock(string);
}

